src/pfb/exporters/tsv.py

```diff
@@ -6,6 +6,14 @@
 
 from ..cli import to_command
 
+plural_parents = {
```

Comment:
PLURAL_PARENTS all caps underscore naming convention for a const module global

Reply:
Should this be configurable rather than a constant global? is this data dictionary specific / have we handled all the cases we know about?

Reply:
These parents node names are part of the minimal data model. Any nodes that are down stream from these top level nodes do not have any problems. I believe it should be fine to just have this as a global constant.